plugin/sign/signer.go

```diff
@@ -58,7 +58,8 @@ func (s *Signer) Sign(now time.Time) (*file.Zone, error) {
 	ln := len(names)
 
 	for _, pair := range s.keys {
-		rrsig, err := pair.signRRs([]dns.RR{z.Apex.SOA}, s.origin, ttl, inception, expiration)
+		var rrsig *dns.RRSIG
+		rrsig, err = pair.signRRs([]dns.RR{z.Apex.SOA}, s.origin, ttl, inception, expiration)
```

Comment:
this is because err is shadow? I don't know I think govet is totally going off the rails with these - its more lines of code for no benefit?